protocol/0090-VAMM-automated_market_maker.md

```diff
@@ -127,6 +127,18 @@ In addition to amending to reduce the size a user may also cancel their AMM enti
 
 Note that, whilst an `Abandon Position` transaction immediately closes the AMM a `Reduce-Only` transaction will keep it active for an uncertain amount of time into the future. During this time, any Amendment transactions received should move the AMM out of `Reduce-Only` mode and back into standard continuous operation.
 
+### Oracle-driven base price
+
+The base price can either be a fixed scalar specified by a user or an [oracle configuration](./0045-DSRC-data_sourcing.md). It should be possible to switch between the two with a regular vAMM amendment transaction.
```

Comment:
I guess its implied, but I'm assuming when a new oracle price comes in to change the base price, the lower and upper boundary prices are also changed by the same delta?

Reply:
First iteration: unless we hear strongly that someone needs the bunds to move as well, let's keep these fixed.

Reply:
@wwestgarth is one of them much cleaner than the other from implementation point of view?

Reply:
It doesn't make any difference to the implementation really, so its mostly what would be most useful to people.

Reply:
Went with fixed bounds for now, let's give people the option to either leave them fixed or updated them proportionally once they ask for it.